qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -1,19 +1,46 @@
 - pci_bridge: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     login_timeout = 360
+    type = pci_bridge
+    only i440fx pseries
+    no usb2
```

Comment:
There have one patch to remove this variants  https://github.com/avocado-framework/avocado-vt/pull/1958.
Just checked that patch, it is ok for me.   Since that patch still have not merged, let's wait a while, i guess we can remove this line then.

Reply:
That patch already be merged,  let's remove `no usb2` here.

Reply:
Updated, thanks!